test/framework/build.gradle

```diff
@@ -47,7 +47,6 @@ tasks.named('forbiddenApisMain').configure {
 // TODO: should we have licenses for our test deps?
 tasks.named("dependencyLicenses").configure { enabled = false }
 tasks.named("dependenciesInfo").configure { enabled = false }
-tasks.named("dependenciesGraph").configure { enabled = false }
```

Comment:
I wonder if this should be part of the snyk check or not. It seems we upload this somewhere so I would consider it kind of production code